treeherder/perf/stats.py

```diff
@@ -315,6 +293,58 @@ def interpret_effect_size(delta):
         return "large"
 
 
+def interpret_cles_direction(cles, pvalue_threshold=PVALUE_THRESHOLD):
+    greater_rev = None
+    if cles is None:
+        return "CLES cannot be interpreted", greater_rev
+    if cles > pvalue_threshold:
+        greater_rev = "base"
+        return f"{cles:.0%} chance a base value > a new value", greater_rev
+    if cles < pvalue_threshold:
+        greater_rev = "new"
+        return f"{1 - cles:.0%} chance a new value > base value", greater_rev
+    return "CLES cannot be interpreted", greater_rev
+
+
+def is_new_better(c_delta, cles, mann_pvalue, lower_is_better, pvalue_threshold=PVALUE_THRESHOLD):
+    """This method takes in CLES to measure if meaningful, Mann Whitney p-val for significance as well as Cliff's Delta for change"""
+    # Possibility Base > than New with a small amount or more significance
+    cles_interpretation, greater_rev = interpret_cles_direction(
+        cles, pvalue_threshold=PVALUE_THRESHOLD
+    )
+    effect_size = interpret_effect_size(c_delta)
```

Comment:
We've already determined this in the code that calls `is_new_better` (it's the line just before the one that calls `is_new_better`. We can reuse that result here.